R/geom_lineribbon.R

```diff
@@ -200,8 +198,8 @@ GeomLineribbon = ggproto("GeomLineribbon", Geom,
     # use case of fit lines / curves: draw the ribbons in order from largest mean width to
     # smallest mean width, so that the widest intervals are on the bottom.
     ribbon_grobs = ribbon_grobs[order(-map_dbl_(ribbon_grobs, `[[`, "width"))] %>%
-      map("grobs") %>%
-      reduce(c)
+      lapply(function(x) x[["grobs"]]) %>%
+      Reduce(c, .)
```

Comment:
It's possible (I'm not 100% sure) that this could be changed to the following, would have to test it.

```suggestion
      unlist(recursive = FALSE)
```

I think it would also be faster (the things I didn't know when I wrote this...).